src/main/java/uk/gov/ons/ssdc/supporttool/service/CaseService.java

```diff
@@ -39,6 +40,9 @@
   @Value("${queueconfig.update-sample-sensitive-topic}")
   private String updateSampleSensitiveTopic;
 
+  @Value("${queueconfig.update-sample-topic}")
+  private String updateSampleTopic;
```

Comment:
what is this? why is it in case processor? update sample is an event which is published _to_ the case processor, not _from_ the case processor